src/github.com/couchbase/sync_gateway/rest/changes_api.go

```diff
@@ -188,6 +188,7 @@ func (h *handler) sendSimpleChanges(channels base.Set, options db.ChangesOptions
 		}
 
 		encoder := json.NewEncoder(h.response)
+		var forceClose bool = false
```

Comment:
if `forceClose := false` works here, I think it should be used.  It's more concise and more commonly seen in Go code